api/tests/src/api/opportunities_v0_1/test_opportunity_route_search.py

```diff
@@ -10,335 +9,124 @@
     FundingInstrument,
     OpportunityStatus,
 )
-from src.db.models.opportunity_models import (
-    Opportunity,
-    OpportunityAssistanceListing,
-    OpportunitySummary,
-)
 from src.util.dict_util import flatten_dict
 from tests.conftest import BaseTestClass
-from tests.src.db.models.factories import (
-    CurrentOpportunitySummaryFactory,
-    LinkOpportunitySummaryApplicantTypeFactory,
-    LinkOpportunitySummaryFundingCategoryFactory,
-    LinkOpportunitySummaryFundingInstrumentFactory,
-    OpportunityAssistanceListingFactory,
-    OpportunityFactory,
-    OpportunitySummaryFactory,
+from tests.src.api.opportunities_v0_1.conftest import (
```

Comment:
Git does not like trying to diff this file. Probably best to look at it on its own to see the current structure as this is a mess.